### PR TITLE
Replaced DeveloperErrors with Check for PolygonPipeline.js

### PR DESCRIPTION
**Pull request by rms13**
Worked on issue #4794, replaced DeveloperError with Check in Source/Core/PolygonPipeline.js

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks @rms13!  @omh1280 do you want to review this?  In general, please jump in a review any PR that you want.

File: Source/Core/PolygonPipeline.js [hunk 1]
```diff
@@ -41,12 +41,8 @@ define([
      */
     PolygonPipeline.computeArea2D = function(positions) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(positions)) {
-            throw new DeveloperError('positions is required.');
-        }
-        if (positions.length < 3) {
-            throw new DeveloperError('At least three positions are required.');
-        }
+        Check.typeOf.object('positions', positions);
```
Comment (omh1280) [inline Source/Core/PolygonPipeline.js]: looks like `positions` is just an array, so use `Check.defined('positions', positions)`

**Comment by rms13** [comment]
@omh1280 
Fixed. Thanks for the correction! 
See you tomorrow!

**Comment by omh1280** [comment]
Looks good!

**Comment by pjcozzi** [comment]
Thanks @rms13 @omh1280!

**Pull request merged**